resources/js/Pages/Auth/ForgotPassword.vue

```diff
@@ -28,8 +28,12 @@
       </div>
 
       <div class="flex items-center justify-end mt-4">
-        <v-btn :loading="form.processing">
-          Email Password Reset Link
+        <v-btn
+          :loading="form.processing"
+          dark
```

Comment:
do we need the `dark` attribute? if the button color is `primary` will it make the text white?
please explain why we need this
thanks